components/ipfs/test/BUILD.gn

```diff
@@ -11,6 +11,13 @@ source_set("brave_ipfs_unit_tests") {
 
   sources = [ "//brave/components/ipfs/ipfs_utils_unittest.cc" ]
 
+  if (!is_android && !is_ios) {
+    sources += [
+      "//brave/components/ipfs/ipfs_component_cleaner.cc",
```

Comment:
why are you adding a source here instead of a dep on the target for that source? This should also have a new buildflag for `enable_ipfs_component` or similar

Reply:
Created follow up issue for that: https://github.com/brave/brave-browser/issues/41626